### DISCUSSION TITLE
Booting SCCM vis HTTP

### DISCUSSION THREAD

**Discussion by jamaya77** [ipxe/ipxe#296] [Q&A] [answered]
Hello,

I've been trying to follow instructions for booting  up with the sccm boot iso, but I'm having trouble configuring the iPXE Boot Menu with the .ipxe script:

https://ipxe.org/howto/sccm

<img width="424" alt="Screen Shot 2021-03-15 at 11 08 49 PM" src="https://user-images.githubusercontent.com/80473349/111250338-8cb9f680-85e3-11eb-81b5-759d3a945c12.png">

I tried configured the boot menu based on the documentation, but I receive an error:

<img width="333" alt="Screen Shot 2021-03-15 at 11 15 39 PM" src="https://user-images.githubusercontent.com/80473349/111250910-92640c00-85e4-11eb-914b-f83b8bd828d3.png">

<img width="821" alt="Screen Shot 2021-03-15 at 10 53 41 PM" src="https://user-images.githubusercontent.com/80473349/111250946-aad42680-85e4-11eb-8719-6ebcd743ac0b.png">

Suggestions?

**Comment by NiKiZe** [comment, marked as answer]
There is no filename command, maybe you want chain instead? Https://iPXE.org/cmd/chain

**Comment by jamaya77** [comment]
Thank you, @NiKiZe , for the correction. I made the changes (using "chain") and it appears to help start the script, but I ran into an error.

<img width="835" alt="Screen Shot 2021-03-16 at 1 37 34 PM" src="https://user-images.githubusercontent.com/80473349/111356904-31c5e500-865f-11eb-80f7-59c60431d77f.png">


sccm.ipxe script:

#!ipxe

kernel wimboot
initrd bootmgr  bootmgr
initrd bcd      BCD
initrd boot.sdi boot.sdi
initrd boot.wim boot.wim
boot

**Reply by NiKiZe** [reply]
Please try with only boot.wim
Then take a screenshoot that includes the first part of iPXE boot.

Also try to follow the error messages, the URLs are there for a reason.

**Comment by jamaya77** [comment]
Sorry the late reply. The boot.wim file appears to load fine from the script, however, a tftp error is displaying which I can't figure out where this tftp call is coming from to boot ubuntu. Is there a way to include in the script to disregard and continue on error? 

The tftp error also occurs when I boot an iso via http.

![tftperror](https://user-images.githubusercontent.com/80473349/111860237-b09b7600-891c-11eb-9717-c7888b3c89b2.png)

**Reply by NiKiZe** [reply]
You should probably debug your scripts more, http://iPXE.org/scripting

**Reply by jamaya77** [reply]
Hey guys. Thanks for your time.
I've corrected the problem with the boot menu calling for run ubuntu via tftp. So now I have the sccm ipxe script runs without interruption. However, it's still not booting to the sccm screen and it's not displaying any errors either. After the boot.wim file loads, it just goes to a black screen with the cursor blinking at the top left corner.

Boot.ipxe:
<img width="359" alt="Screen Shot 2021-03-23 at 10 04 59 PM" src="https://user-images.githubusercontent.com/80473349/112243583-36bd0280-8c24-11eb-9e5d-361420fbd3f7.png">


sccm.ipxe:
![Screen Shot 2021-03-23 at 9 44 02 PM](https://user-images.githubusercontent.com/80473349/112242955-1b052c80-8c23-11eb-9c9c-24c557811a62.png)

**Comment by jamaya77** [comment]
Guys,
Just wanted to confirm that the issue is resolved. I copied over the whole wimboot directory instead of just the "winboot" file within directory. I appreciate all your help. Thanks.
